config/SupportedVehicles.xml

```diff
@@ -96,6 +96,33 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.018 1.39 1.352" width="2.195" height="4.48" length="7.78"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_920/rpx920.xml">
+            <loadingArea offset="0.095 1.39 0.816" width="2.195" height="4.48" length="8.662"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_1226/rpx1226.xml">
+            <loadingArea offset="0.059 1.39 -0.657" width="2.195" height="4.48" length="11.831"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
+            <loadingArea offset="-0.02 0.978 -2.661" width="1.991" height="2.642" length="5.268"/>
```

Comment:
Loading area could use a little more length if we wanted. Very optional though:

![image](https://user-images.githubusercontent.com/15822357/165582969-bfb4879e-c003-42f0-a09d-a629ce90a69b.png)

You can see above the loading zone comes a bit short of where the ramp starts.

Reply:
Same with width, there's probably room for a little more if we want:

![image](https://user-images.githubusercontent.com/15822357/165583272-f65624fc-45ab-4365-833f-cede6c98ae3a.png)

Reply:
Suggestion add ```<options enableRearLoading="true"/>``` if you want to allow rear autoloading via pallet forks.